### PR TITLE
Chore/merge location into org

### PR DESCRIPTION
**Pull request by DominikGuzei**
In the end it was a big refactoring but i think it's worth it :smiley:
Now locations are part of orgs, both in the domain as well as in projections / denormalized orgs collection.

### CODE REVIEW

File: packages/domain/source/server/appeals/pledge.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Space.domain.Entity.extend(Donations, 'Pledge', {
 
   STATES: {
-    new: `new`,
+    'new': `new`,
```
Comment (rhyslbw) [inline packages/domain/source/server/appeals/pledge.js:4]: Is this a mistake?

**Comment by rhyslbw** [comment]
I agree, it makes the Organisation aggregate much more interesting. we've dropped 50 lines of code in the process too :-)

**Comment by rhyslbw** [comment]
the projection is also pooch

File: packages/domain/source/server/appeals/pledge.js (hunk 1 above)
Comment (DominikGuzei) [inline packages/domain/source/server/appeals/pledge.js:4]: no this is an error in eslint – you have to quote javascript keywords

Comment (rhyslbw) [inline packages/domain/source/server/appeals/pledge.js:4]: ah right, got it

**Pull request merged**